spec/services/notify_app_store/message_builder_spec.rb

```diff
@@ -64,7 +64,7 @@
             'town' => 'Lawyer Town'
           },
           'first_hearing_date' => '2023-03-01',
-          'gender' => nil,
+          'gender' => 'm',
```

Comment:
so these are the values -> they should be mapped to translations (I think)

Reply:
These will need to be added to another ticket for mapping in that case